boskos/projects

```diff
@@ -0,0 +1,3 @@
+PROJECT-FAKE-1
```

Comment:
What are the restrictions on this file? We should have a test that loads it and ensures they're valid project names.